[2021-03-04 21:07:31,285] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T21:07:26.948897+00:00 [queued]>
[2021-03-04 21:07:31,332] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T21:07:26.948897+00:00 [queued]>
[2021-03-04 21:07:31,332] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 21:07:31,333] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-03-04 21:07:31,333] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 21:07:31,397] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): load_data> on 2021-03-04T21:07:26.948897+00:00
[2021-03-04 21:07:31,400] {standard_task_runner.py:54} INFO - Started process 6127 to run task
[2021-03-04 21:07:31,539] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'clean_data', 'load_data', '2021-03-04T21:07:26.948897+00:00', '--job_id', '60', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_clean.py', '--cfg_path', '/tmp/tmpmlgnfu7q']
[2021-03-04 21:07:31,544] {standard_task_runner.py:78} INFO - Job 60: Subtask load_data
[2021-03-04 21:07:31,634] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: clean_data.load_data 2021-03-04T21:07:26.948897+00:00 [running]> 6623e80cd770
[2021-03-04 21:07:31,667] {data_clean.py:65} INFO - ****** NOAH starting load data
[2021-03-04 21:07:31,705] {logging_mixin.py:112} WARNING - /home/airflow/.local/lib/python3.6/site-packages/pandas/core/generic.py:2612: UserWarning: The spaces in these column names will not be changed. In pandas versions < 0.14, spaces were converted to underscores.
  method=method,
[2021-03-04 21:07:31,909] {data_clean.py:77} INFO - ****** NOAH 2 ending load data
[2021-03-04 21:07:31,911] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-03-04 21:07:31,920] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=clean_data, task_id=load_data, execution_date=20210304T210726, start_date=20210304T210731, end_date=20210304T210731
[2021-03-04 21:07:36,219] {local_task_job.py:102} INFO - Task exited with return code 0
